radiography/src/main/java/radiography/RenderTreeString.kt

```diff
@@ -75,7 +76,7 @@ private fun <N> StringBuilder.renderRecursively(
   var newLastChildMask = lastChildMask
   for (index in 0..lastChildIndex) {
     if (index == lastChildIndex) {
-      newLastChildMask = newLastChildMask or (1 shl depth).toLong()
+      newLastChildMask = (newLastChildMask.clone() as BitSet).apply { set(depth) }
```

Comment:
This is pretty gross. Got a better idea?

Reply:
Since this isn't exposed, seems ok to use a mutable bitset. We just need to set and then unset.

Reply:
Derp, that's much better.

Reply:
Done.